src/app/components/MediaPlayer/types.d.ts

```diff
@@ -0,0 +1,63 @@
+export type PlayerConfig = {
+  product?: string;
+  superResponsive: boolean;
+  counterName?: string;
+  playlistObject: {
+    title: string;
+    holdingImageURL: string;
+    items: Item[];
+  };
+  statsObject?: { clipPID?: string };
+};
+
+export type Item = {
+  versionID: string;
+  kind: string;
+  duration: number;
+  live?: boolean;
+};
+
+export type Player = {
+  load: () => void;
+};
+
+export type BumpType = {
+  player: (div: HTMLDivElement | null, config: PlayerConfig) => Player;
+};
+
+export type MediaBlock = {
+  type: string;
+  model: Partial<{
+    locator: string;
+    text: string;
+    title: string;
+    blocks: MediaBlock[];
+    imageUrl: string;
+    versions: {
+      versionId: string;
+      duration: number;
+      warnings?: { [key: string]: string };
+    }[];
+    smpKind: string;
+  }>;
+};
+
+export type Props = {
+  blocks: MediaBlock[];
+};
+
+export type BuildConfigProps = {
+  id: string | null;
+  pageType: string;
+  blocks: MediaBlock[];
+  counterName: string | null;
+};
+
+declare global {
```

Comment:
Do we usually put the global types elsewhere? https://github.com/bbc/simorgh/blob/857f0ba17f31d6b34b7821ed3307f256319d1de9/src/global.d.ts